crypto/fipsmodule/cipher/e_aes.c

```diff
@@ -902,6 +959,20 @@ DEFINE_METHOD_FUNCTION(EVP_CIPHER, EVP_aes_256_ofb) {
   out->cipher = aes_ofb_cipher;
 }
 
+DEFINE_METHOD_FUNCTION(EVP_CIPHER, EVP_aes_256_wrap) {
+  memset(out, 0, sizeof(EVP_CIPHER));
+
+  out->nid = NID_id_aes256_wrap;
+  out->block_size = 8;
+  out->key_len = 32;
+  out->iv_len = 8;
+  out->ctx_size = sizeof(EVP_AES_WRAP_CTX);
+  out->flags = EVP_CIPH_WRAP_MODE | EVP_CIPH_CUSTOM_IV |
+               EVP_CIPH_FLAG_CUSTOM_CIPHER | EVP_CIPH_ALWAYS_CALL_INIT;
```

Comment:
Do you need to handle `EVP_CIPH_CUSTOM_IV` in `EVP_CipherInit_ex()` for `EVP_CIPH_WRAP_MODE`?